backend/src/index.js

```diff
@@ -129,11 +129,8 @@ async function main() {
       if (wamp.session) {
         const dataStats = await aggregateStats();
         const { transactions, blocks } = await queryDashboardBlocksAndTxs();
-        wampPublish(
-          "database-stats",
-          [{ dataStats, transactions, blocks }],
-          wamp
-        );
+        wampPublish("database-stats", [{ dataStats }], wamp);
+        wampPublish("database-list", [{ transactions, blocks }], wamp);
```

Comment:
Please, spend more time on naming things as it is hard to review and maintain.

database stats is somewhat meaningful, but database list has no reasonable meaning to me. I still suggest considering the high-level picture and see that `database-stats` is hard to reason about while `network-stats` or `chain-stats` are more specific, and later we may add `economics-stats`.

P.S. Whenever you change semantics, make sure that the parent function/class/module names are still describing the implementation accurately.